docs/visualize/timelion.asciidoc

```diff
@@ -504,3 +504,52 @@ image::images/timelion-conditional04.png[]
 {nbsp}
 
 For additional information on Timelion conditional capabilities, go to https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+[float]
+[[timelion-deprecation]]
+=== Timelion App deprecation
+
+In *8.0* and later, the Timelion app - deprecated since 7.0 - is going to be removed. There are actions you need to complete to copy your existing Timelion worksheets to a dashboard.  
```

Comment:
Deprecated since 7.0, the Timelion app will be removed in 8.0.  If you have any Timelion worksheets, you must migrate them to a dashboard.